pallets/gear-builtin/src/lib.rs

```diff
@@ -85,6 +87,148 @@ pub type GasAllowanceOf<T> = <<T as Config>::BlockLimiter as BlockLimiter>::GasA
 const LOG_TARGET: &str = "gear::builtin";
 pub type ActorErrorHandleFn = HandleFn<BuiltinContext, BuiltinActorError>;
 
+#[derive(Clone, Copy, Default, Eq, PartialEq, TypeInfo)]
+#[cfg_attr(feature = "std", derive(serde::Serialize, serde::Deserialize))]
+pub struct BuiltinActorId {
```

Comment:
what about moving this stuff along with BuiltinActorType to the gbuiltins?
Ids are demanded by sails